Changes/updates to all html files and css file, specifically for grid.html

Em-A-C opened the pull request:
After adding grid.html to this branch, I updated the navigation on all my HTML pages to add "Grid" as a nav bar option. I then updated the CSS to create a grid based on the divs in grid.html. I wrote all new code for grid.html in a CodePen and then copied it over to my file here, because it made it easier to visualize how I was styling the grid.

Em-A-C committed:
Added grid.html file from week 3's assignment

Em-A-C committed:
Added grid.html to navigation

Em-A-C committed:
Added grid.html to navigation

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update grid.html

Em-A-C committed:
Added styling for grid.html

I typed the CSS in a CodePen so I could layout the grid easier, and then I copied/pasted into this GitHub file.

Em-A-C committed:
Updated the organization of the divs and how the text inside the divs was presented.

Em-A-C committed:
Update styles.css

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update styles.css from what I had worked on in a CodePen

Em-A-C committed:
Had to add a comment explaining why I decided to use hsl color codes instead of hex or rgb color codes

Em-A-C committed:
Update grid.html

dselicaro commented:
Hi Em -- I do see a lot of code added to your CSS for grid but the changes are not taking effect. See what you can do to fix and resubmit this pull request to your Week 3 assignment.

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update styles.css

Em-A-C committed:
Rename styles.css to styles.scss

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update index.html

Em-A-C committed:
Update contact.html

Em-A-C commented:
I think its not rendering properly because I used SCSS instead of plain CSS. I changed the file extension from CSS to SCSS, but I am still not for sure its rendering correctly. On top of that, I think my link is not in the correct format. Do you have any suggestions of how I could fix these problems?

Em-A-C committed:
Update link to scss again

Em-A-C committed:
Updated scss link again

Em-A-C committed:
updates scss link again

Em-A-C committed:
Added classes to the paragraph elements to hopefully make a clearer separation between global styles and the styles for grid.html

Em-A-C committed:
Updated p element for grid.html styling

Em-A-C committed:
Rename styles.scss to styles.css

Em-A-C committed:
Updated styles.css

Re-did my CSS so it didnt include SCSS. I just got too frustrated with it and have been fighting with it for two days so I gave up.

Em-A-C committed:
Update contact.html

Em-A-C committed:
Update grid.html

Em-A-C committed:
Update index.html

Em-A-C committed:
Update styles.css

Em-A-C committed:
Update styles.css
